Apps/WebClient/src/ClientApp/src/views/timeline.vue

```diff
@@ -683,6 +687,67 @@ export default class TimelineView extends Vue {
             });
     }
 
+    private fetchEncounters() {
+        // Placeholder to test until service is complete
+        this.isEncounterLoading = true;
+        this.timelineEntries.push(
+            new EncounterTimelineEntry({
+                id: "8bcbc841-899f-486a-a4b9-9b5172dcb70a",
+                practitionerName: "Smith, David",
+                specialtyDescription: "Doctor",
+                serviceDateTime: new Date(),
+                clinic: {
+                    name: "Best Clinic Ever",
+                    addressLine1: "1122 Fake St. V8N2C8",
+                    clinicId: "38138e65-d657-4ce1-8b52-1f51f8695cb5",
+                    phoneNumber: "2508582268",
+                },
+            })
+        );
+        this.isEncounterLoading = false;
+        // const encounterService: IEncounterService = container.get(
```

Comment:
Do we have a azure ticket to fix this later?

Reply:
adding a "wire it up" ticket

Reply:
having a mock encounter service would be good for now instead of commenting it out

Reply:
I always have the same question. Should we get a end-to-end integration task in our next stories for this kind of work?